Library/Homebrew/test/language/python_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "language/python"
+require "resource"
+
+describe Language::Python::Virtualenv::Virtualenv do
+  subject { described_class.new(formula, dir, "python") }
+
+  let(:dir) { Pathname.new(Dir.mktmpdir) }
+
+  let(:resource) { double("resource", stage: true) }
+  let(:formula_bin) { dir/"formula_bin" }
+  let(:formula) { double("formula", resource: resource, bin: formula_bin) }
+
+  after(:each) do
+    dir.rmtree
```

Comment:
If `dir` hasn't been used in a test (unlikely but possible in future), this will create the directory just to immediately destroy it.